### PR TITLE
Update Chinese document：auto-detect mode and crossorigin

### PR DESCRIPTION
**Pull request by warriorBrian**
Update Chinese document：auto-detect mode and crossorigin
Fix the description of the error
😸

### CODE REVIEW

File: zh/api/configuration-modern.md [hunk 1]
```diff
@@ -30,4 +30,15 @@ description: Build and server a modern bundle
...
+
+- 当未指定`modern`时，在打包时Nuxt将自动检测`nuxt start`中的`modern`，自动检测模式为：
+
+| Mode          | Modern Mode   |
+| ------------- |:-------------:|
+| universal     | server        |
+| spa           | client        |
+
+- 在使用`nuxt generate`时，`modern mode`只能是`client`
+- 使用 [`build.crossorigin`](/api/configuration-build#crossorigin) 请在`<link>` 和 `<script>` 中设置 `crossorigin` 属性
```
Comment (clarkdo) [inline zh/api/configuration-modern.md]: 请在可能有点歧义😊

Comment (warriorBrian) [reply zh/api/configuration-modern.md]:
```suggestion
- 使用 [`build.crossorigin`](/api/configuration-build#crossorigin) 在`<link>` 和 `<script>` 中设置 `crossorigin` 属性
```

File: zh/api/configuration-modern.md [hunk 2]
```diff
@@ -3,23 +3,23 @@ title: "API: modern 属性"
...
 
-> 此功能的想法来自 [vue-cli modern mode](https://cli.vuejs.org/guide/browser-compatibility.html#modern-mode) 
+> 此功能的想法来自 [vue-cli modern mode](https://cli.vuejs.org/guide/browser-compatibility.html#modern-mode)
 
 - 类型: `String` 或 `Boolean`
   - 默认: false
   - 可能的值:
-    - `'client'`: 同时提供现代捆绑打包`<script type ="module">`和旧捆绑打包`<script nomodule>脚本，同时为现代捆绑打包提供`<link rel ="modulepreload">`。每个了解模块类型的浏览器都会加载现代软件包，而旧版浏览器则会回归到旧版浏览器（已编译）。
-    - `'server'` or `true`: Node.js服务器将根据用户代理检查浏览器版本，并提供相应的现代或旧版捆绑。
+    - `'client'`: 同时提供现代浏览器捆绑依赖打包`<script type ="module">`和兼容性低浏览器捆绑依赖打包`<script nomodule>脚本，同时为现代捆绑打包提供`<link rel ="modulepreload">`。每个了解模块类型的浏览器都会加载现代软件包，而旧版浏览器则会回归到旧版浏览器（已编译）。
```
Comment (warriorBrian) [inline zh/api/configuration-modern.md]:
```suggestion
    - `'client'`: 构建两个版本的包：同时提供面向支持现在浏览器的 ES2015+ 写法支持 `<script type ="module">` 和兼容性低的其他旧浏览器的包 `<script nomodule>` 的脚本，同时为现代浏览器打包提供 `<link rel ="modulepreload">` 。每个正确解析模块类型的现代浏览器都会加载现代浏览器软件依赖包，而旧版浏览器则会加载到指定旧版浏览器的依赖包（已编译）。
```

**Comment by clarkdo** [comment]
🙏

**Pull request merged**